test/integration/search.spec.js

```diff
@@ -30,9 +30,13 @@ describe('Search', () => {
       expect(this.resourceList.map(i => i.summary)).to.eql(resourcesListModel.map(i => i.summary))
     );
     it('should display correct categories for all resources', () =>
-      expect(this.resourceList.map(i => i.categories))
+      expect(this.resourceList.map(i => i.categories.map(j => j.text)))
         .to.eql(resourcesListModel.map(i => i.category))
     );
+    it('should contain correct link to all categories for all resources', () =>
+      expect(this.resourceList.map(i => i.categories.map(j => j.link)))
```

Comment:
Would be good to include which test is running in the title of the test otherwise the failing case may be difficult to find.